### ISSUE TITLE
Unable to upload report using coverage-reporter v0.6.8

### ISSUE THREAD

**Issue by ogenstad** [coverallsapp/coverage-reporter#119]
### What was wrong?

I'm not able to upload the report using the 0.6.8 version that was just released.

### Logs

```bash
Run coverallsapp/github-action@v2
Run mkdir -p ~/bin/
coveralls-linux.tar.gz: OK
Run coveralls report        
 
⠀⠀⠀⠀⠀⠀⣿
⠀⠀⠀⠀⠀⣼⣿⣧⠀⠀⠀⠀⠀⠀⠀ ⣠⣶⣾⣿⡇⢀⣴⣾⣿⣷⣆ ⣿⣿⠀⣰⣿⡟⢸⣿⣿⣿⡇ ⣿⣿⣿⣷⣦⠀⠀⢠⣿⣿⣿⠀⠀⣿⣿⠁⠀⣼⣿⡇⠀⢀⣴⣾⣿⡷
⠶⣶⣶⣶⣾⣿⣿⣿⣷⣶⣶⣶⠶  ⣸⣿⡟ ⠀⢠⣿⣿⠃⠈⣿⣿⠀⣿⣿⢠⣿⡿⠀⣿⣿⣧⣤⠀⢸⣿⡇⣠⣿⡿⠀⢠⣿⡟⣿⣿⠀⢸⣿⡿⠀⠀⣿⣿⠃⠀⢸⣿⣧⣄
⠀⠀⠙⢻⣿⣿⣿⣿⣿⡟⠋⠁⠀⠀ ⣿⣿⡇⠀ ⢸⣿⣿⠀⣸⣿⡟⠀⣿⣿⣾⡿⠁ ⣿⣿⠛⠛⠀⣿⣿⢿⣿⣏⠀⢀⣿⣿⣁⣿⣿⠀⣾⣿⡇⠀⢸⣿⡿⠀⠀⡀⠙⣿⣿⡆
⠀⠀⢠⣿⣿⣿⠿⣿⣿⣿⡄⠀⠀⠀ ⠙⢿⣿⣿⠇⠈⠿⣿⣿⡿⠋⠀⠀⢿⣿⡿⠁⠀⢸⣿⣿⣿⡇⢸⣿⣿⠀⣿⣿⣄⣾⣿⠛⠛⣿⣿⢠⣿⣿⣿ ⣼⣿⣿⣿ ⣿⣿⡿⠋⠀
⠀⢀⣾⠟⠋⠀⠀⠀⠙⠻⣷⡀⠀⠀
 
  v0.6.8

🔍 Detected coverage file: coverage.xml
🔍 Detected coverage file: .coverage
#<CoverageReporter::CoveragepyParser::ParserError:--: 1: coverage: not found
>
```

As we're still waiting for https://github.com/coverallsapp/github-action/issues/170 I don't know if I have any options now using the github-action. 

Please let me know if I need to provide any additional information around this, or if I can do something on my end to fix it.

**Comment by pjhoberman** [comment]
We're seeing this as well with our circleci integration.

**Comment by matthieudesprez** [comment]
Same here using latest circleci orb coveralls/coveralls@2.2.1

**Comment by jacekbj** [comment]
Isn't `coverage` now required indirectly? 

https://github.com/coverallsapp/coverage-reporter/commit/ac469acf9bbc239ef70900d87bdef7689a5dd488#diff-b07b49e9438f2f536ecc020f2c360f6afd54e56006bd7aa2b59cfff0a6338369R33

Tests don't fail because `coverage` is installed explicitly in `ci.yml`.

**Comment by pjhoberman** [comment]
> Isn't `coverage` now required indirectly?

We have coverage in our requirements, and I just tested via shell to ensure coverage is present on the box and it is. 
```
circleci@31e6e1eee119:~/project$ ./coveralls report --debug
 
⠀⠀⠀⠀⠀⠀⣿
⠀⠀⠀⠀⠀⣼⣿⣧⠀⠀⠀⠀⠀⠀⠀ ⣠⣶⣾⣿⡇⢀⣴⣾⣿⣷⣆ ⣿⣿⠀⣰⣿⡟⢸⣿⣿⣿⡇ ⣿⣿⣿⣷⣦⠀⠀⢠⣿⣿⣿⠀⠀⣿⣿⠁⠀⣼⣿⡇⠀⢀⣴⣾⣿⡷
⠶⣶⣶⣶⣾⣿⣿⣿⣷⣶⣶⣶⠶  ⣸⣿⡟ ⠀⢠⣿⣿⠃⠈⣿⣿⠀⣿⣿⢠⣿⡿⠀⣿⣿⣧⣤⠀⢸⣿⡇⣠⣿⡿⠀⢠⣿⡟⣿⣿⠀⢸⣿⡿⠀⠀⣿⣿⠃⠀⢸⣿⣧⣄
⠀⠀⠙⢻⣿⣿⣿⣿⣿⡟⠋⠁⠀⠀ ⣿⣿⡇⠀ ⢸⣿⣿⠀⣸⣿⡟⠀⣿⣿⣾⡿⠁ ⣿⣿⠛⠛⠀⣿⣿⢿⣿⣏⠀⢀⣿⣿⣁⣿⣿⠀⣾⣿⡇⠀⢸⣿⡿⠀⠀⡀⠙⣿⣿⡆
⠀⠀⢠⣿⣿⣿⠿⣿⣿⣿⡄⠀⠀⠀ ⠙⢿⣿⣿⠇⠈⠿⣿⣿⡿⠋⠀⠀⢿⣿⡿⠁⠀⢸⣿⣿⣿⡇⢸⣿⣿⠀⣿⣿⣄⣾⣿⠛⠛⣿⣿⢠⣿⣿⣿ ⣼⣿⣿⣿ ⣿⣿⡿⠋⠀
⠀⢀⣾⠟⠋⠀⠀⠀⠙⠻⣷⡀⠀⠀
 
  v0.6.8

🔍 Detected coverage file: .coverage
☝️ Detected coverage format: python - .coverage
⏱️ Report parsing: 00:00:00.052937362
Unhandled exception: --: 1: coverage: not found
 (CoverageReporter::CoveragepyParser::ParserError)
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
circleci@31e6e1eee119:~/project$ pip list
Package                           Version     Editable project location
--------------------------------- ----------- --------------------------
...
coverage                          7.4.4
...
```

**Comment by afinetooth** [comment]
All,

Thanks for the reports. We're working on an update that we hope to release shortly. I'll come back with an update.

Thanks.

**Comment by mike-burns** [comment]
I've reverted the coverage.py change that is leading to these errors, and cut release 0.6.9.

I'm closing this issue, but please comment here if it's still a problem.

**Comment by jan-janssen** [comment]
`0.6.10` has the same bug: 
```
Run coverallsapp/github-action@v2
Run mkdir -p ~/bin/
coveralls-linux.tar.gz: OK
Run coveralls report        
 
⠀⠀⠀⠀⠀⠀⣿
⠀⠀⠀⠀⠀⣼⣿⣧⠀⠀⠀⠀⠀⠀⠀ ⣠⣶⣾⣿⡇⢀⣴⣾⣿⣷⣆ ⣿⣿⠀⣰⣿⡟⢸⣿⣿⣿⡇ ⣿⣿⣿⣷⣦⠀⠀⢠⣿⣿⣿⠀⠀⣿⣿⠁⠀⣼⣿⡇⠀⢀⣴⣾⣿⡷
⠶⣶⣶⣶⣾⣿⣿⣿⣷⣶⣶⣶⠶  ⣸⣿⡟ ⠀⢠⣿⣿⠃⠈⣿⣿⠀⣿⣿⢠⣿⡿⠀⣿⣿⣧⣤⠀⢸⣿⡇⣠⣿⡿⠀⢠⣿⡟⣿⣿⠀⢸⣿⡿⠀⠀⣿⣿⠃⠀⢸⣿⣧⣄
⠀⠀⠙⢻⣿⣿⣿⣿⣿⡟⠋⠁⠀⠀ ⣿⣿⡇⠀ ⢸⣿⣿⠀⣸⣿⡟⠀⣿⣿⣾⡿⠁ ⣿⣿⠛⠛⠀⣿⣿⢿⣿⣏⠀⢀⣿⣿⣁⣿⣿⠀⣾⣿⡇⠀⢸⣿⡿⠀⠀⡀⠙⣿⣿⡆
⠀⠀⢠⣿⣿⣿⠿⣿⣿⣿⡄⠀⠀⠀ ⠙⢿⣿⣿⠇⠈⠿⣿⣿⡿⠋⠀⠀⢿⣿⡿⠁⠀⢸⣿⣿⣿⡇⢸⣿⣿⠀⣿⣿⣄⣾⣿⠛⠛⣿⣿⢠⣿⣿⣿ ⣼⣿⣿⣿ ⣿⣿⡿⠋⠀
⠀⢀⣾⠟⠋⠀⠀⠀⠙⠻⣷⡀⠀⠀
 
  v0.6.10

🔍 Detected coverage file: .coverage
⚠️ Coverage reporter does not yet know how to process this file: .coverage
🚨 Nothing to report
Error: Process completed with exit code 1.
```

**Comment by pjhoberman** [comment]
We're also experiencing this issue again. I opened a new issue: https://github.com/coverallsapp/coverage-reporter/issues/124

**Comment by afinetooth** [comment]
All, just doing some housekeeping:

- @pjhoberman I am following up on your issue in #124 
- @jan-janssen aere you still experiencing any issues? I see in #124 that you resolved your issue by using `coverage xml` (converting `coverage.py`'s `.coverage` output file to `coverage.xml` and using the `cobertura` parser with `format: cobertura` (or `--format cobertura`). That is in fact our recommendation because that non-standard-format `.coverage` file (a SQLite3 database) is hard to parse and our beta parser was struggling with it. We've updated our `pytest-cov` parser since to use `coverage` from your environment and perform that same conversion (via `coverage xml`), then send it through our `cobertura` (XML) parser.

**Comment by jan-janssen** [comment]
@afinetooth The `coverage xml` command fixed the issue for me, so I am fine.
